packages/contracts/deploy/new/40_finalize-managing-dao/40_revoke-permissions.ts

```diff
@@ -4,6 +4,7 @@ import {getContractAddress, managePermissions, Permission} from '../../helpers';
 import {Operation} from '../../../utils/types';
 import {PluginRepo__factory} from '../../../typechain';
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DAO__factory} from '../../../typechain/osx-versions/v1_2_0/contracts/core/dao/DAO.sol';
```

Comment:
Etc

Reply:
Fixed in https://github.com/aragon/osx/commit/1ac77f580109dd5c4612e7a70899fae68135606f